drivers/flash/flash_stm32_qspi.c

```diff
@@ -684,6 +944,7 @@ static int flash_stm32_qspi_init(const struct device *dev)
 	dma_cfg.linked_channel = STM32_DMA_HAL_OVERRIDE;
 	ret = dma_config(dev_data->dma.dev, dev_data->dma.channel, &dma_cfg);
 	if (ret != 0) {
+		LOG_ERR("Confuring DMA");
```

Comment:
Typo in Confuring, and this probably should have been LOG_ERR("%d: Failed to configure DMA", ret);